velox/functions/lib/string/StringImpl.h

```diff
@@ -173,6 +173,64 @@ std::vector<int32_t> stringToCodePoints(const T& inputString) {
   return codePoints;
 }
 
+template <typename TInString>
+FOLLY_ALWAYS_INLINE int64_t
```

Comment:
Why are these functions templatized when they are only being invoked with string_view anyways ? It would be simpler to just pass string_view parameters for left and right in these functions.

Reply:
Thanks @aditi-pandit .

I just followed the convention maintained in this header file. All the input strings are templatized. I am also having thought.